tcl/auth.tcl

```diff
@@ -76,7 +76,7 @@ proc qc::auth_session { session_id } {
     #| Try to authenticate user based on the session_id given
     #| Return the user_id if successful
     #| On failure throw AUTH error
-    if { [session_exists $session_id] } {
+    if { [session_valid $session_id] } {
         return [session_user_id $session_id]
```

Comment:
Likely not what is wanted for everything. However simply checking existence won't work because the session could be expired.

Not sure how we can get around this without having to change all calls to `auth_session`.